HACKING.md

```diff
@@ -299,3 +310,38 @@ http://simple-httpd-fedora-coreos.$CLUSTER_URL
 ```
 
 (Or simply check the output of `oc get route simple-httpd`).
+
+
+
+## Running A Developer Pipeline Alongside an Existing Deployment
+
+If you already know of an existing deployed pipeline you can add some
+namespaced (i.e. DEVEL_PREFIX) resources into the existing project
+and run your own developer pipeline. This is useful if you simply want
+to test a different COSA image or FCOS config or pipeline code, but
+otherwise maintain the manifest the same.
+do this, you *must* use the `devel-up` script. For example:
```

Comment:
Non sequitur

Reply:
yeah - copy/paste mistake I think - deleted in next upload.